tests/core/test_channels.py

```diff
@@ -417,6 +417,49 @@ def test_socketio_channel():
         assert text["text"] == "Hi there!"
 
 
+def test_botframework_attachments():
```

Comment:
nice test! can you also add test for the case where you supply a `metadata` dict?

Reply:
I've added a test, let me know what you think about it

Reply:
looks great 👍